src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -342,6 +342,7 @@ def load_arguments(self, _):
                    help='Indicate whether Automatic Updates is enabled for the Windows virtual machine')
         c.argument('patch_mode', arg_type=get_enum_type(self.get_models('InGuestPatchMode')), min_api='2020-06-01',
                    help='Mode of in-guest patching to IaaS virtual machine. Possible values are: Manual - You  control the application of patches to a virtual machine. You do this by applying patches manually inside the VM. In this mode, automatic updates are disabled; the paramater --enable-auto-update must be false. AutomaticByOS - The virtual machine will automatically be updated by the OS. The parameter --enable-auto-update must be true. AutomaticByPlatform - the virtual machine will automatically updated by the OS. The parameter --enable-agent and --enable-auto-update must be true')
+        c.argument('ssh_key_name', help='SSH key name')
```

Comment:
if we finally decide to keep the feature of creating a ssh key through `az vm create` command, I suggest we show users more detailed description of this argument.

We need tell users, when this parameter work with `--generate-ssh-keys` or `--ssh-key-values`, it means we gonna create a ssh key, and when it appears alone, it means we are expecting an existing ssh key name.

Anyway, I wish we WON'T keep the feature...